Disable NGiNX buffering for v2 Registry

fancyremarker opened the pull request:
@krallin : Can/should I be writing a test for this? If you have a suggestion for a worthwhile test here, please let me know and I'll take a stab at implementing.

krallin commented:
I think the current tests should be fine @fancyremarker. It looks like the build is failing here, though, want me to take a look at what happened? I don't think it's related to your change — it's likely we have a dependency somewhere that isn't pinned (whereas it _should_ be pinned).

fancyremarker commented:
Thanks @krallin ; I think the build failure was the same issue I fixed in ea543fd (part of open PR #5). I rebased on top of that PR and the build passed. #5 should also be mergeable if it's OK by you.

krallin commented:
Thanks @fancyremarker. I thought we had merged #5 already..! Let's merge both now.
